.github/workflows/beam_PreCommit_ItFramework.yml

```diff
@@ -83,6 +83,13 @@ jobs:
         requires-py-38: false
         requires-py-39: false
         requires-go: false
+    - name: Authenticate on GCP
```

Comment:
Are these needed by the tests? If I am not wrong, the credentials are only needed for IT's/LT's, which aren't run in this PreCommit?

Reply:
Yes, it is needed: it appears resource managers still needs credential setup

Failed run (without credential): https://github.com/Abacn/beam/actions/runs/5682012376/job/15399424990

successful run (with Authenticate on GCP step):

https://github.com/Abacn/beam/actions/runs/5682497039/job/15401009175

Reply:
Ah, gotcha